### DISCUSSION TITLE
我无法上传我的学生信息

### DISCUSSION THREAD

**Discussion by Lehcok** [community/community#20298] [GitHub Education]
I take a picture with my camera and it always says the school ID and date are on the photo, I have all the required information on my photo, but why isn't it recognized?
![Screenshot_2022-07-11-08-30-46-197_com android chrome-edit](https://user-images.githubusercontent.com/63954316/178168647-bdd9113c-3e80-4b82-af9e-1cbf3a8cd06a.jpg)

**Comment by Tostisto** [comment]
You can use [GitHub Education](https://support.github.com/contact/education)

**Reply by LingYunNetWork** [reply]
勇啊 用中文标题 建议使用英语
我也遇到了这样的问题 我的中考成绩单提交了上去 但是没有过
原因是因为他们无法识别中文
但是他们给的回答很敷衍：

![image](https://user-images.githubusercontent.com/108985539/178248370-e0f6be9c-7fc9-46e3-b1c2-bdc86f9e394e.png)

**Reply by Tostisto** [reply]
Then try to rewrite to English.

**Comment by LingYunNetWork** [comment]
我建议 你可以造假 伪造一张英语的证明（也许这样有用？？？
同样 建议尽可能地规范你的英语语法（防止被嫌弃），这个时候 谷歌翻译会很好用
